### PR TITLE
Use a recursive countdown method

### PR DESCRIPTION
**Pull request by NightlyNexus**

### CODE REVIEW

File: telecine/src/main/java/com/jakewharton/telecine/OverlayView.java [hunk 1]
```diff
@@ -162,23 +155,33 @@ private void startRecording() {
   }
 
   private void showCountDown() {
-    recordingView.setText(R.string.countdown_three);
-    postDelayed(new Runnable() {
-      @Override public void run() {
-        recordingView.setText(R.string.countdown_two);
-        postDelayed(new Runnable() {
+    String[] countdown = getResources().getStringArray(R.array.countdown);
+    if (countdown.length == 0) {
```
Comment (JakeWharton) [inline telecine/src/main/java/com/jakewharton/telecine/OverlayView.java]: When would this ever happen?

File: telecine/src/main/java/com/jakewharton/telecine/OverlayView.java [hunk 2]
```diff
@@ -162,23 +155,33 @@ private void startRecording() {
...
+      countdownComplete();
+    }
+    countdown(countdown, 0);
```
Comment (JakeWharton) [inline telecine/src/main/java/com/jakewharton/telecine/OverlayView.java]: Missing an `else`, if the `if` is actually something that's needed...

File: telecine/src/main/java/com/jakewharton/telecine/OverlayView.java (hunk 1 above)
Comment (NightlyNexus) [inline telecine/src/main/java/com/jakewharton/telecine/OverlayView.java]: Yeah, good point.  We should crash anyway when the array is empty.

**Comment by JakeWharton** [comment]
Thanks

**Pull request merged**